go.mod

```diff
@@ -201,7 +201,7 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/nats-io/nats-server/v2 v2.9.21
 	github.com/nats-io/nats.go v1.28.0
-	github.com/openvex/go-vex v0.2.1
+	github.com/openvex/go-vex v0.2.5
```

Comment:
this is a breaking change that will need remediation

Reply:
Looks like right now, we only use csaf, but #1241 will likely need changes.

Reply:
Agreed, just did a review and while the spec v0.2.0 change is breaking, it doesn't impact this

Reply:
I will fix this and push the changes.

Reply:
This should be fixed by: https://github.com/guacsec/guac/pull/1306